### PR TITLE
Added nanoTag

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/nanoThings/nanoTag/uplink.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      break;
+    }
+    case 26:
+    case 27: {
+      topic = "report_frame";
+      data.fid = Bits.bitsToUnsigned(bits.substr(0, 16));
+
+      let i = 1;
+      for (let bytePos = 16; bytePos < bits.length; bytePos += 16) {
+        data[`temperature${i}`] =
```
Comment (mariusgiger) [inline types/nanoThings/nanoTag/uplink.js:50]: rather than doing this, output multiple samples?

File: types/nanoThings/nanoTag/uplink.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          Math.round(
+            (Bits.bitsToUnsigned(bits.substr(bytePos, 16)) / 100 - 50) * 10,
+          ) / 10;
+        i++;
+      }
+      break;
+    }
+    case 22:
+      topic = "first_timestamp";
+      data.firstSampleTimestamp = Bits.bitsToUnsigned(bits.substr(0, 32));
```
Comment (mariusgiger) [inline types/nanoThings/nanoTag/uplink.js:60]: convert this to a Date?

File: types/nanoThings/nanoTag/uplink.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      topic = "device_status";
+      lifecycle.batteryVoltage = Bits.bitsToUnsigned(bits.substr(0, 16)) / 1000;
+      data.lastSampleNumber = Bits.bitsToUnsigned(bits.substr(16, 32));
+      data.lastSampleTimestamp = Bits.bitsToUnsigned(bits.substr(32, 32));
+      break;
+    case 31:
+      lifecycle.batteryVoltage = Bits.bitsToUnsigned(bits.substr(0, 16)) / 1000;
+      break;
+    default:
+      topic = "unknown";
```
Comment (mariusgiger) [inline types/nanoThings/nanoTag/uplink.js:74]: output the full payload?

File: types/nanoThings/nanoTag/uplink.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        data[`temperature${i}`] =
+          Math.round(
+            (Bits.bitsToUnsigned(bits.substr(bytePos, 16)) / 100 - 50) * 10,
+          ) / 10;
+        i++;
+      }
+      break;
+    }
+    case 22:
+      topic = "first_timestamp";
```
Comment (mariusgiger) [inline types/nanoThings/nanoTag/uplink.js:59]: what is the purpose of this topic?

**Review by mariusgiger — COMMENTED** [review]
looks good, a few small things to clarify - after this the PR can be merged

**Review by sevyakenza — APPROVED** [review]

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**